scripts/deploy

```diff
@@ -31,9 +31,15 @@ var Uploader = function () {
   this.secrets = new Secrets();
   this.secrets.load();
   this.s3 = new AWS.S3();
+  this.tag = this.getPkgVersion();
   this.distAssets = fs.readdirSync(DIST_DIR);
 };
 
+Uploader.prototype.getPkgVersion = function () {
+  var pkg = fs.readFileSync('./package.json')
```

Comment:
You can actually do `require('./package.json')` I think. Which will remove the need to manually parse json on the next line.